sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -248,7 +248,7 @@
       "get": {
         "summary": "Get the identities of multiple devices from the IoT hub identity registry. Not recommended. Use the IoT Hub query language to retrieve device twin and device identity information. See https://docs.microsoft.com/en-us/rest/api/iothub/service/queryiothub and https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-query-language for more information.",
         "description": "Get the identities of multiple devices from the IoT hub identity registry. Not recommended. Use the IoT Hub query API to retrieve device twin and device identity information. See https://docs.microsoft.com/en-us/rest/api/iothub/service/queryiothub and https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-query-language for more information.",
-        "operationId": "RegistryManager_GetDevices",
+        "operationId": "Devices_GetDevices",
```

Comment:
Did we not want this to be Devices_Get? Or is that going to be a conflict with modules?

Reply:
Module has it's own group, this needs to be Device_Get in that case

Reply:
+1 to Abhipsa's point. Seems redundant to me

Reply:
I thought about it earlier and don't have strong opinion here. If we want to change, we can discuss sure.